.github/ISSUE_TEMPLATE/backport_release.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Backport Release
+about: Checklist for Backport releases
+
+---
+
+<!--
+Please check items along as you follow the release process.
+-->
+
+#### Overview
+
+This is a checklist for backport releases. This is filled in by both the releaser and the reviewer where necessary.
+
+#### Preparation
+
+- [ ] Create a release/<version> branch off the backport/<minor> branch
+```bash
```

Comment:
I think this needs some extra indentation so that it is in the li instead of under. Same below.